compiler/aarch64/codegen/TreeEvaluatorTable.hpp

```diff
@@ -394,21 +394,21 @@
     TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::loadaddrEvaluator ,	// TR::loadaddr		// load address of non-heap storage item (Auto; Parm; Static or Method)
     TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::ZEROCHKEvaluator ,	// TR::ZEROCHK		// Zero-check an int.  Symref indicates call to perform when first child is zero.  Other children are arguments to the call.
     TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::callIfEvaluator ,	// TR::callIf		// Call symref if first child evaluates to true.  Other childrem are arguments to the call.
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::iRegLoadEvaluator ,	// TR::iRegLoad		// Load integer global register
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::aRegLoadEvaluator ,	// TR::aRegLoad		// Load address global register
-    TR::TreeEvaluator::unImpOpEvaluator ,        // TODO:ARM64: Enable when Implemented: TR::TreeEvaluator::lRegLoadEvaluator ,	// TR::lRegLoad		// Load long integer global register
+    TR::TreeEvaluator::iRegLoadEvaluator, // TR::iRegLoad		// Load integer global register
+    TR::TreeEvaluator::aRegLoadEvaluator, // TR::aRegLoad		// Load address global register
+    TR::TreeEvaluator::lRegLoadEvaluator, // TR::lRegLoad		// Load long integer global register
```

Comment:
Why not save the extra level of indirection and have this table reference `commonRegLoadEvaluator` instead, and delete the i,a,l, etc. versions?